installers/install.ps1

```diff
@@ -6,12 +6,14 @@ install.ps1 -b branchToInstall
 
 Set-StrictMode -Off
 
+# URL to fetch update infos from.
+$script:BASEINFOURL = "https://platform.activestate.com/sv/state-update/api/v1/info"
 # URL to fetch installer archive from
 $script:BASEFILEURL = "https://state-tool.s3.amazonaws.com/update/state"
 # The name of the remove archive to download
 $script:ARCHIVENAME = "state-installer.zip"
 # Name of the installer executable to ultimately use
-$script:INSTALLERNAME = "state-installer.exe"
+$script:INSTALLERNAME = "state-install\\state-installer.exe"
```

Comment:
I'm trying to understand why this changed but I can't figure it out, can you explain? Is the install archive for "latest" structured differently than that of a specific version?

Reply:
The archive pointed to by platform.activestate.com has a slightly different directory structure than the archive we were hardcoded to look for.